src/transforms/mapperTransforms.ts

```diff
@@ -264,10 +276,8 @@ function getAdditionalProperties(allParents: Schema[]): Mapper | undefined {
 }
 
 function isUberParent(objectSchema: ObjectSchema) {
```

Comment:
I've always been a little unclear on the uber parent. Is this just any ObjectSchema that has no parents but is extended?

Reply:
Yes, that's an uber parent

Reply:
I'm adding that to the description when we extract Uber Parents so that it is clear